dashboard/app/views/programming_expressions/show.html.haml

```diff
@@ -1,5 +1,9 @@
+//%script{src: webpack_asset_path('js/googleblockly.js')}
+%script{src: webpack_asset_path('js/blockly.js')}
```

Comment:
perhaps a question for the star labs team: when should we switch to google blockly? could be worth capturing in comments here or on `use_google_blockly`

Reply:
my understanding is that we can use google blockly here if the corresponding level type uses google blockly (which can be found out from a level type's response to [uses_google_blockly?](https://github.com/code-dot-org/code-dot-org/blob/9eb532455dd844e9c9446d40ffa94aaab40795c6/dashboard/app/models/levels/flappy.rb#L86-L88)). however, i don't think i understand how possible/easy that info is to gather from this part of the code. @ajpal for a second opinion -- is my understanding correct?

Reply:
Yeah, unfortunately this isn't connected to a specific level. I've been working on the assumption that for a given block pool, all blocks will use google blockly or none, but I don't know if that's a good assumption. So, right now, we only have spritelab documentation, which uses code.org blockly (I think?) but in the future this should work for poetry which uses google blockly (again, I think?) by switching which script gets loaded.

Reply:
I think that is a fair assumption- it's hard to imagine a use-case for mixing blockly versions within a block pool.
Not all blocks are part of a pool though. Older labs have blocks defined in JS files, not on LB so there is no block pool.

Reply:
> Not all blocks are part of a pool though. Older labs have blocks defined in JS files, not on LB so there is no block pool.

Good point. We do have docs for a few blocks defined in our blockly repo, but I guess then using code-dot-org blockly for those is the right choice.